docs/develop/integrations/vercel-integration.md

```diff
@@ -30,9 +30,11 @@ After pressing the `Create` button you'll see the new `chat` cache in the list o
 
 ![Chat cache](/img/console-caches-chat.png)
 
-Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure that you choose the same region when you generate a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, choose the cloud provider and region you used to create your cache, choose the `Super User Token` token type, and hit the `Generate Token` button.
+Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure that you choose the same region when you generate a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, choose the cloud provider and region you used to create your cache. Since the cache is already created, we will use a fine grained token which will allow the worker to read from and write to the cache; but will not allow it to do control plane operations such as delete. This is especially helpful if you want to manage security of control plane and data plane operations separately.
```

Comment:
```suggestion
Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure you choose the same region when generating a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, and choose the cloud provider and region you used to create your cache. Since the cache is already created, we will use a fine-grained token that will allow the worker to read from and write to the cache; but will not allow it to do control plane operations such as delete. This is especially helpful if you want to manage security of control plane and data plane operations separately.
```